### PR TITLE
Render coverage information into a new page instead of modal window.

### PR DESCRIPTION
**Pull request by robertsipka**
Also prepare the coverage component to show detailed information of each file.

### CODE REVIEW

File: src/components/device/coverage/coverage-table/coverage-table.component.js [hunk 1]
```diff
@@ -75,6 +75,12 @@ export default class CoverageTable extends React.Component {
             </span>
           ),
           accessor: "fileName",
+          Cell: e =>
```
Comment (rtakacs) [inline src/components/device/coverage/coverage-table/coverage-table.component.js]: Could you use a more informative name?

File: src/components/device/coverage/coverage.component.js [hunk 2]
```diff
@@ -35,29 +70,68 @@ export default class Coverage extends React.Component {
   }
 
   render() {
-    const { project, device, load, error } = this.props;
+    const { project, device, location, load, error } = this.props;
 
-    if (project.key !== 'iotjs' || device.key === 'stm32f4dis' || load || error) {
+    if ((project.key !== 'iotjs' || device.key === 'stm32f4dis') && !load) {
```
Comment (rtakacs) [inline src/components/device/coverage/coverage.component.js]: What about `project.key === 'jerryscript'` ?

File: src/components/device/coverage/coverage.component.js [hunk 3]
```diff
@@ -35,29 +70,68 @@ export default class Coverage extends React.Component {
...
+    const coverage = this.getCoverage();
+    const commit = this.getCommitInfo(coverage);
+    const lineInfo = this.getLineInfo(coverage);
+    const color = this.getBadgeColor(lineInfo.percentage);
+    const coverageTable = coverage[0].coverage_info;
 
-          <div className="btn btn-link" data-toggle="modal" data-target="#coverage-info">
-            <FontAwesomeIcon icon="chart-area" />
-            <span className="ml-2">Coverage</span>
+    const fileName = location.substring(location.lastIndexOf('/')+1);
```
Comment (rtakacs) [inline src/components/device/coverage/coverage.component.js]: ` + 1`

**Comment by robertsipka** [comment]
@rtakacs , @LaszloLango : Thanks for the reviews. I've updated it accordingly.

**Pull request merged**